images/installer.yml.in

```diff
@@ -29,6 +30,9 @@ volumes:
 - name: mkimage-raw-efi
   image: MKRAW_TAG
   readonly: true
+- name: uboot
```

Comment:
Isn't it ARM specific? Shouldn't we add it conditionally somehow?

Reply:
Nah, it is available for all platforms, even riscv64. Different content. It works on all of them.

Reply:
Strange... Why do we need u-boot for x86, for example... But ok, let it be...